scss/_settings_spacing.scss

```diff
@@ -8,44 +8,44 @@ $sp-unit-ratio: 0.5 !default;
 $sp-unit: 1rem * $sp-unit-ratio !default;
 
 $font-sizes: (
-  h1: pow($ms-ratio, 8),
-  h1-mobile: pow($ms-ratio, 6),
-  h2: pow($ms-ratio, 6),
-  h2-mobile: 1.83274,
-  h3: pow($ms-ratio, 4),
-  h3-mobile: pow($ms-ratio, 3),
-  h4: pow($ms-ratio, 2),
-  h4-mobile: 1.22176,
+  h1: 2.5,
+  h1-mobile: 2.5,
+  h2: 2.5,
+  h2-mobile: 2.5,
+  h3: 1.5,
+  h3-mobile: 1.5,
+  h4: 1.5,
+  h4-mobile: 1.5,
   small: pow($ms-ratio, -1),
   x-small: pow($ms-ratio, -2),
 ) !default;
 
 $line-heights: (
-  h1: 7 * $sp-unit,
+  h1: 6 * $sp-unit,
   h1-mobile: 6 * $sp-unit,
   h2: 6 * $sp-unit,
-  h2-mobile: 5 * $sp-unit,
-  h3: 5 * $sp-unit,
+  h2-mobile: 6 * $sp-unit,
+  h3: 4 * $sp-unit,
   h3-mobile: 4 * $sp-unit,
   h4: 4 * $sp-unit,
-  h4-mobile: 3 * $sp-unit,
+  h4-mobile: 4 * $sp-unit,
   default-text: 3 * $sp-unit,
   small: 2.5 * $sp-unit,
   x-small: 2 * $sp-unit,
 ) !default;
 
 // baseline nudges for type scale ratio of (16/14)^2
 $nudges: (
-  h1-large: 0.15rem,
-  h1-mobile: 0.165rem,
-  h1: 0.2rem,
-  h2: 0.2rem,
-  h2-mobile: 0.1rem,
-  h3: 0.1rem,
-  h3-mobile: 0.5rem,
-  h4-large: 0,
-  h4: 0.05rem,
-  h4-mobile: 0.3rem,
+  h1-large: 0.5rem,
+  h1-mobile: 0.5rem,
+  h1: 0.5rem,
+  h2: 0.5rem,
+  h2-mobile: 0.5rem,
+  h3: 0.45rem,
+  h3-mobile: 0.45rem,
+  h4-large: 0.45rem,
+  h4: 0.45rem,
```

Comment:
@lyubomir-popov This change is causing a huge shift. From almost nothing, to 0.45rem it's pushing all h4s (and related headings in components) almost whole spacing unit lower.

How should we compensate for that? Move everything else lower as well (icons that should align with that?

Reply:
Yes, the good thing is that all nudges are roughly the same value - so padding on top of text becomes more consistent.